packages/neuron-ui/src/components/VerifyHardwareAddress/index.tsx

```diff
@@ -139,7 +141,7 @@ const VerifyHardwareAddress = ({ address, wallet, onDismiss }: VerifyHardwareAdd
     const res = await getDevicePublicKey()
     if (isSuccessResponse(res)) {
       const { result } = res
-      if (result?.address === address) {
+      if (result?.address === address || result?.address === shortAddress) {
```

Comment:
`shortAddress` could be generated in the callback to avoid an extra prop of this Component